### DISCUSSION TITLE
Supported Types of Codes

### DISCUSSION THREAD

**Discussion by SebastianA** [juliansteenbakker/mobile_scanner#865] [General]
Hi,
Unfortunately, I couldn't find an answer to my question about which codes are supported. Barcodes, QR codes or also Data Matrix codes?

Thanks
Sebastian

**Comment by hermanniscoding** [comment]
You can find the supported codes here:
https://github.com/juliansteenbakker/mobile_scanner/blob/91bb85b8808a34f80a87d897424762de7ab1250d/lib/src/enums/barcode_format.dart
